clients/python/setup.py

```diff
@@ -18,9 +18,27 @@
 ###############################################################################
 
 import os
-import re
+
 from setuptools import setup, find_packages
 
+
+def prerelease_local_scheme(version):
+    """Return local scheme version unless building on master in CircleCI.
+    This function returns the local scheme version number
+    (e.g. 0.0.0.dev<N>+g<HASH>) unless building on CircleCI for a
+    pre-release in which case it ignores the hash and produces a
+    PEP440 compliant pre-release version number (e.g. 0.0.0.dev<N>).
+    """
+
+    from setuptools_scm.version import get_local_node_and_date
+
+    if 'CIRCLE_BRANCH' in os.environ and \
+       os.environ['CIRCLE_BRANCH'] == 'master':
```

Comment:
if you wanted to avoid the backslash, this could just be 

```python
if os.environ.get('CIRCLE_BRANCH') == 'master':

Reply:
Changed to `os.getenv`.